CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.4.30 (Feb 2021)
+  * [GH-4619](https://github.com/JetBrains/kotlin-native/pull/4619) Fix NSUInteger size for Watchos x64
```

Comment:
I guess it makes sense to use the link to the issue this PR fixes: https://youtrack.jetbrains.com/issue/KT-44083

Reply:
ok